lldb/source/Plugins/Language/CPlusPlus/Coroutines.cpp

```diff
@@ -199,10 +199,12 @@ lldb_private::formatters::StdlibCoroutineHandleSyntheticFrontEnd::Update() {
   return lldb::ChildCacheState::eRefetch;
 }
 
-size_t StdlibCoroutineHandleSyntheticFrontEnd::GetIndexOfChildWithName(
+llvm::Expected<size_t>
+StdlibCoroutineHandleSyntheticFrontEnd::GetIndexOfChildWithName(
     ConstString name) {
   if (!m_resume_ptr_sp || !m_destroy_ptr_sp)
-    return UINT32_MAX;
+    return llvm::createStringError("Cannot find index of child '%s'",
```

Comment:
Lets print the pointer values of `m_resume_ptr_sp` and `m_destroy_ptr_sp` here

Reply:
Fixed, thanks!